ansible/roles_ocp_workloads/ocp4_workload_example/tasks/post_workload.yml

```diff
@@ -26,4 +24,4 @@
     msg: "Post-Software checks completed successfully"
   when:
   - not silent|bool
-  - workload_shared_deployment|default(False)
\ No newline at end of file
+  - workload_shared_deployment|default(False)
```

Comment:
This should be:

```
  - workload_shared_deployment | default(false) | bool
```

Not something the yamllint would catch, but will come up with ansible linting